### ISSUE TITLE
Inactive thumbnails for "paging: continuous"

### ISSUE THREAD

**Issue by triplingual** [ProjectMirador/mirador#3293]
With an LTR scroll using `paging: continuous`, thumbnails are generated and conjoined appropriately but clicking on them doesn't navigate the scroll, at any zoom level.

Example:
[IIIF Recipe](https://preview.iiif.io/cookbook/0011-book-3-behavior/recipe/0011-book-3-behavior/), and [relevant manifest in Mirador](https://projectmirador.org/embed/?iiif-content=https://preview.iiif.io/cookbook/0011-book-3-behavior/recipe/0011-book-3-behavior/manifest-continuous.json)

**Comment by dnoneill** [comment]
I don't think it is supposed to. There is one "thumbnail" for scroll. You can tell because there is outline on the entire image. You are on the one image. I don't think the thumbnail view is supposed to be a zoom/interactive view. TL;DR not a bug, probably needs UX review on the feature.
